its/projects/NuGet.config

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+      <add key="nuget.org" value="https://api.nuget.org/v3/index.json" protocolVersion="3" />
```

Comment:
We also had a discussion with Kabeel on the topic of .NET UTs, where we use nuget.org and he said we should migrate to Repox there too.
https://github.com/SonarSource/sonar-dotnet/issues/8665

So this should use Repox, username and password, like the rest.